features/admin.extensions.v1/configs/application.tsx

```diff
@@ -224,11 +224,11 @@ export const applicationConfig: ApplicationConfig = {
         roleMapping: true
     },
     customApplication: {
-        allowedProtocolTypes: [
-            SupportedAuthProtocolTypes.OAUTH2_OIDC,
-            SupportedAuthProtocolTypes.SAML,
-            SupportedAuthProtocolTypes.WS_FEDERATION
-        ],
+        allowedProtocolTypes: (): string[] => {
```

Comment:
shall we update the config name to something like `getAllowedProtocolTypes` since this is now a function?